src/ripple/app/tests/Flow_test.cpp

```diff
@@ -977,7 +977,7 @@ struct Flow_test : public beast::unit_test::suite
 
         auto const timeDelta = Env{*this}.closed ()->info ().closeTimeResolution;
 
-        for(auto const& t :{dcSoTime(), numAccSoTime()}){
+        for(auto const& t :{amendmentRIPD1141SoTime(), amendmentRIPD1141SoTime()}){
```

Comment:
You're right; That's not needed now that I just have one switchover over rather than two. Fixed.